Libraries/LibJS/Runtime/ArrayPrototype.cpp

```diff
@@ -447,6 +448,65 @@ Value ArrayPrototype::reduce(Interpreter& interpreter)
     return accumulator;
 }
 
+Value ArrayPrototype::reduceRight(Interpreter& interpreter)
```

Comment:
Please follow the [Serenity C++ coding style](https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#names) and use snake_case for function names.

Reply:
sorry! force of habit :)